pkgs/development/tools/misc/luarocks/default.nix

```
@@ -19,6 +19,11 @@ stdenv.mkDerivation rec {
   };
 
   patches = [ ./darwin-3.1.3.patch ];
+
+  postPatch = lib.optionalString stdenv.hostPlatform.isDarwin ''
+    substituteInPlace src/luarocks/core/cfg.lua --subst-var-by 'darwinMinVersion' '${stdenv.hostPlatform.darwinMinVersion}'
```

```suggestion
  postPatch = lib.optionalString stdenv.targetPlatform.isDarwin ''
    substituteInPlace src/luarocks/core/cfg.lua --subst-var-by 'darwinMinVersion' '${stdenv.targetPlatform.darwinMinVersion}'
```
because we should do this for other -> darwin cross too, I believe?

---

I don't know if nixpkgs has has working cross compilation for lua packages, but assuming it does, `targetPackages` seems correct here 👍.